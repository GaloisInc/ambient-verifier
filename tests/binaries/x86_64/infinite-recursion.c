/*
 * A test for which simulation should not terminate unless a recursion bound
 * is imposed.
 */

void loop() {
  loop();
}

int main() {
  loop();
  return 0;
}
