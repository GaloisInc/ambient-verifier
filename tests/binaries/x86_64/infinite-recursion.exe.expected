successful: 0
failed: 1
recursionBound: 10
